draft-regext-rdap-extensions.md

```diff
@@ -534,7 +558,7 @@ other properly registered extension.
 
 [@!RFC9083] notes that the extension identifiers provide a "hint" to the client
 as to how to interpret the response. This wording does not intentionally restrict
-the extension to defining only JSON values within the extensions namespace.
+the extension to defining only JSON values within the extensions' namespace.
```

Comment:
Did we mean "extension's namespace" here since we talk of other extensions later.